models/news.model.js

```diff
@@ -0,0 +1,35 @@
+const db  = require("../db/connection");
+
+const readTopics = () => {
+  return db.query("SELECT * FROM topics")
+    .then((result) => {
+     return result.rows;
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+};
+
+const readArticles = (sort_by = 'created_at') => {
+
+  const acceptedSorBy = ["created_at"];
+  if (!acceptedSorBy.includes(sort_by)){
+    return Promise.reject({status: 400, msg: 'Please sort by acceptable parameter'})
```

Comment:
This is neither required for the endpoint **at the moment** AND hasn't been tested for. TDD remember and only add what the ticket asks for!